### PR TITLE
Data refresh behavior 

### PR DESCRIPTION
**Pull request by daniel-tabarcea**
Moved javascript source code to a behavior so it can be used without the markup

### CODE REVIEW

File: etools-refresh-behavior.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    clearLocalStorage: function() {
+      localStorage.clear();
+    },
+
+    clearDexieDbs: function() {
+      // except Chrome and Opera this method will delete only the dbs created with Dexie
+      Dexie.getDatabaseNames(function(dbsNames) {
+        this.dexieDbsNumber = dbsNames.length;
+        if (this.dexieDbsNumber > 0) {
+          dbsNames.forEach(function(dbName) {
```
Comment (adi130987) [inline etools-refresh-behavior.html:42]: @daniel-tabarcea could we refactor this and use `Promise.all` for better check when all dexie dbs were deleted. By the time this was implemented I missed this option that fits better with this functionality. What do you think?

File: etools-refresh-behavior.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../etools-dexiejs/etools-dexiejs.html">
```
Comment (adi130987) [inline etools-refresh-behavior.html]: There is a new `dexie.html` file I created not log ago. We can use it to import DexieJS library. No need to use `etools-dexiejs.html` element anymore.

File: etools-refresh-behavior.html (hunk 1 above)
Comment (daniel-tabarcea) [reply etools-refresh-behavior.html:42]: It's a good idea, I'll do it.

Comment (adi130987) [reply etools-refresh-behavior.html:42]: It's not a priority right now, I've created an issue and will make the improvement later.

**Review by adi130987 — APPROVED** [review]

**Pull request merged**